contrib/node/src/python/pants/contrib/node/subsystems/resolvers/npm_resolver.py

```diff
@@ -69,6 +98,7 @@ def _emit_package_descriptor(self, node_task, target, results_dir, node_paths):
     dependenciesToRemove = [
       'dependencies', 'devDependencies', 'peerDependencies', 'optionalDependencies'
     ]
+    logger.error('Removing dependencies')
```

Comment:
I don't think people are going to understand this error. Would recommend either removing it, or explicitly saying which dependency is being dropped.